src/testIntegration/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/mocks/HmppsAuthMockServer.kt

```diff
@@ -72,6 +72,23 @@ class HmppsAuthMockServer : WireMockServer(WIREMOCK_PORT) {
         )
     )
   }
+  // If the user is not known then this endpoint still returns the username, just not the active caseload
```

Comment:
Ah, hence the nullable `activeCaseloadId` on `UserDetails`?
And this is to support our smoke test user? A bit weird that the `hmpps-auth` 'me' endpoint can return a success response for a user it doesn't know about, but I guess that helps us in our situation of needing to support a smoke test user without creating it in `hmpps-auth` 👍  🤷‍♂️

Reply:
Yes a bit strange. But the endpoint checks tokens that it previously issued so kind of fair enough...